plugins/templates/explanations/closes.jinja2.md

```diff
@@ -0,0 +1 @@
+We use bug prediction in coala which relies on the `Fixes` keyword in commit messages. To get good results from that we need to use `Closes` for normal issues instead of `Fixes` which should only be used for real bugs. (See also [the commit message docs](https://coala.io/commit).) To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.
```

Comment:
Too big for one line, please follow 80 cols per line here as well IMO

Reply:
Same for every template file

Reply:
Add a `LineLengthBear` to `jinja` section of coafile

Reply:
Discussion in gitter regarding this. It messes up the gitter formatting to limit lines to 80 characters.